File: app/src/main/java/com/wealoha/social/beans/location/Location.java
package com.wealoha.social.beans.location;

import java.io.Serializable;

import android.text.TextUtils;

public class Location implements Serializable {

	/**
	 * @Fields serialVersionUID : TODO(用一句话描述这个变量表示什么)
	 */
	private static final long serialVersionUID = -3591352045359882813L;
	public String address;
	public String name;
	public String id;
	public Double latitude;
	public Double longitude;

	/**
	 * @Title: isEmpty
	 * @Description: 地理位置信息是否为空
	 */
	public boolean isEmpty() {
		if (TextUtils.isEmpty(id) || latitude == null || longitude == null) {
			return true;
		}
		return false;
	}
}

File: app/src/main/java/com/wealoha/social/beans/Location.kt
package com.wealoha.social.beans

import android.text.TextUtils
import java.io.Serializable

class Location : Serializable {
    @kotlin.jvm.JvmField
    var address: String? = null
    @kotlin.jvm.JvmField
    var name: String? = null
    @kotlin.jvm.JvmField
    var id: String? = null
    @kotlin.jvm.JvmField
    var latitude: Double? = null
    @kotlin.jvm.JvmField
    var longitude: Double? = null

    /**
     * @Title: isEmpty
     * @Description: 地理位置信息是否为空
     */
    val isEmpty: Boolean
        get() = if (TextUtils.isEmpty(id) || latitude == null || longitude == null) {
            true
        } else false

    companion object {
        /**
         * @Fields serialVersionUID : TODO(用一句话描述这个变量表示什么)
         */
        private const val serialVersionUID = -3591352045359882813L
    }
}